test/common/config/utility_test.cc

```diff
@@ -318,20 +318,21 @@ TEST(UtilityTest, TypedStructToStruct) {
   EXPECT_TRUE(Protobuf::util::MessageDifferencer::Equals(out, untyped_struct));
 }
 
+namespace {
+const std::string bootstrap_config_yaml = R"EOF(
```

Comment:
no global std::string which is not trivial constructable and causes static initialization fiasco. just put in test you use this.

Reply:
done